src/nvim/ops.c

```diff
@@ -6398,6 +6443,22 @@ void do_pending_operator(cmdarg_T *cap, int old_col, bool gui_yank)
       }
       check_cursor_col();
       break;
+    case OP_ATREGREPLAY:
+      if (curwin->w_cursor.lnum + oap->line_count - 1 >
+          curbuf->b_ml.ml_line_count) {
```

Comment:
unnecessary linebreak
```suggestion
      if (curwin->w_cursor.lnum + oap->line_count - 1 > curbuf->b_ml.ml_line_count) {
```